### PR TITLE
Update vision submodule to the latest version, update Changelog

### PR DESCRIPTION
**Pull request by yunikkk**

### CODE REVIEW

**Comment by kiryldz** [comment]
@yunikkk - do we need it in `dev`? Perhaps `master` branch would be better.

**Comment by yunikkk** [comment]
> @yunikkk - do we need it in `dev`? Perhaps `master` branch would be better.

Yeah the next pull will sync dev-master, when the release will be published

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,16 +1,25 @@
...
 
-## v0.10.1
+## 0.12.0
+
+- Added support for non-Snapdragon powered devices. Most chips on the market are supported now,
+including Exynos by Samsung, Kirin by Huawei, Mediatek, etc.
+- Added japanese traffic signs 
+- Improved lanes detection algorithm
+- Improved camera calibration speed
+- Utilized new ML models that reduce resource utilization
```
Comment (kiryldz) [inline CHANGELOG.md:10]: Personally I don't like `utilized`.
IMHO `improved` should be OK here.

Comment (yunikkk) [reply CHANGELOG.md:10]: Well, we haven't improved old models but implemented new ones and then used/utilized them in Teaser. Why don't you like utilized?

Comment (kiryldz) [reply CHANGELOG.md:10]: Let's stick to iOS version.
`- Utilized new ML models that reduce resource consumption`

Comment (yunikkk) [reply CHANGELOG.md:10]: Yeah it's 1:1 already. Moreover - I've suggested this specific wording in iOS request)

Comment (kiryldz) [reply CHANGELOG.md:10]: To be 1:1 it should be `consumption` and not `utilization`.

Comment (yunikkk) [reply CHANGELOG.md:14]: ... душный

Comment (kiryldz) [reply CHANGELOG.md]: 🥇 🥇 🥇

**Comment by yunikkk** [comment]
@torypaulsmith PTAL, this PR updates changelog similar to https://github.com/mapbox/vision-ios-examples/pull/138

**Comment by yunikkk** [comment]
@torypaulsmith merging PR without your approve since it's basically copy of the iOS one, feel free to add comments - will fix them in the new PR

**Pull request merged**